dynawo/3rdParty/adept/adept-chain-2.0.5.sh

```diff
@@ -100,7 +100,7 @@ install_adept() {
   else
     ./configure "CXXFLAGS=-O3 -fPIC $CXX_STD" --prefix=$INSTALL_DIR CC=$DYNAWO_C_COMPILER CXX=$DYNAWO_CXX_COMPILER --disable-openmp $ADEPT_LIBRARY_TYPE_OPTION
   fi
-  make -j $DYNAWO_NB_PROCESSORS_USED && make install
+  make -j $DYNAWO_NB_PROCESSORS_USED && make install && if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" -a "`uname`" = "Darwin" ]; then install_name_tool -id @rpath/libadept.dylib $INSTALL_DIR/lib/libadept.dylib; fi
```

Comment:
I left it on one line to not change 
RETURN_CODE=$?
return ${RETURN_CODE}
because we would need to the check the code of the first two things and then in the case the if is executed check it also, which would be difficult to read also in my sense put it is always possible

Reply:
OK